src/validateNumber.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const NUM_LENGTH = 4;
+
+module.exports.isNumberValid = (number) => {
+  for (let i = 1; i < NUM_LENGTH; i++) {
+    if (number[i] === number[i - 1]) {
+      return false;
+    }
```

Comment:
this redundant but why do you think that the same number will be placed only together